cluster-autoscaler/expander/price/price_test.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package price
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"k8s.io/autoscaler/cluster-autoscaler/expander"
+
+	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider/test"
+	. "k8s.io/autoscaler/cluster-autoscaler/utils/test"
+	apiv1 "k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/plugin/pkg/scheduler/schedulercache"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type testPricingModel struct {
+	nodePrice map[string]float64
+	podPrice  map[string]float64
+}
+
+func (tpm *testPricingModel) NodePrice(node *apiv1.Node, startTime time.Time, endTime time.Time) (float64, error) {
+	if price, found := tpm.nodePrice[node.Name]; found {
+		return price, nil
+	}
+	return 0.0, fmt.Errorf("price for node %v not found", node.Name)
+}
+
+func (tpm *testPricingModel) PodPrice(node *apiv1.Pod, startTime time.Time, endTime time.Time) (float64, error) {
+	if price, found := tpm.podPrice[node.Name]; found {
+		return price, nil
+	}
+	return 0.0, fmt.Errorf("price for pod %v not found", node.Name)
+}
```

Comment:
Can we add a test including nodes and pods not in testPricingModel maps to test error handling?